documentation/2.0/content/userguide/target_env.md

```diff
@@ -24,7 +33,73 @@ This example would apply the `k8s` target type to the discovery result, and plac
 
-### The target configuration file
+### Pre-configured target environments
+
+These target environment configurations are included in the WebLogic Deploy Tooling installation.
+
+#### The WebLogic Kubernetes Operator target
+
+This target environment can be applied by providing the command-line argument `-target wko`. It will provide this additional processing:
+
+- The `wko_operator_filter.py` filter will be applied to remove model elements that are not compatible with the Kubernetes environment
+- Variables will be injected into the model for port, host, and URL attributes
+- `lax` validation will be applied for the resulting model
+- Credentials in the model will be replaced with references to Kubernetes secrets, and a script to create those secrets will be produced
+- An additional Kubernetes resource file, `wko-domain.yaml`, will be produced, with cluster and naming information derived from the model
+
+#### The Verrazzano target
+
+This target environment can be applied by providing the command-line argument `-target vz`. It will provide this additional processing:
+
+- The `vz_filter.py` filter will be applied to remove model elements that are not compatible with the Kubernetes environment
+- Variables will be injected into the model for port, host, and URL attributes
+- `lax` validation will be applied for the resulting model
+- Credentials in the model will be replaced with references to Kubernetes secrets, and a script to create those secrets will be produced
+- One additional Kubernetes resource file, `vz-application.yaml`, will be produced, with cluster and data source information derived from the model
+
+#### Generic Kubernetes target
+
+This target environment can be applied by providing the command-line argument `-target k8s`. It will provide this additional processing:
+
+- The `k8s_operator_filter.py` filter will be applied to remove model elements that are not compatible with the Kubernetes environment
+- Variables will be injected into the model for port, host, and URL attributes
+- `lax` validation will be applied for the resulting model
+- Credentials in the model will be replaced with references to Kubernetes secrets, and a script to create those secrets will be produced
+
+### Using secret credentials in the model
+
+If a target environment is configured to use Kubernetes secrets for credential attribute values, each of those values is replaced with a token using the format `@@SECRET:@@ENV:DOMAIN_UID@@<secret-suffix>:<key>`. For example:
+```yaml
+PasswordEncrypted: '@@SECRET:@@ENV:DOMAIN_UID@@-jdbc-generic1:password@@'
+```
+When a domain is created or updated using a model with these tokens, the environment variable `DOMAIN_UID` should be set to the domain's UID, and secrets with corresponding names should have been created. See [Model Tokens]({{< relref "/concepts/model#model-tokens" >}}) for more details about using secret tokens.
```

Comment:
See [Model Tokens]({{< relref "/concepts/model#model-tokens" >}}) for more details about using secret tokens. -> For more details about using secret tokens, see [Model Tokens]({{< relref "/concepts/model#model-tokens" >}}). (this is the standard Oracle style for providing this type of cross-reference; please make this change globally)